ViAn/ViAn.pro

```diff
@@ -148,7 +152,10 @@ HEADERS += reportgenerator.h\
     Project/Analysis/interval.h \
     Project/Analysis/tagframe.h \
     GUI/Bookmark/myinputdialog.h \
-    GUI/Analysis/anasettingwidget.h
+    GUI/Analysis/anasettingwidget.h \
+    Project/imagesequence.h \
+    Project/video.h \
```

Comment:
Didn't we have this one here before? This isn't a new file.

Reply:
I had some issues when merging in the changes from development. That's probably the reason why